### DISCUSSION TITLE
Stability.ai plugin support

### DISCUSSION THREAD

**Discussion by m-sadegh-sh** [TheFireCo/genkit-plugins#183] [Ideas]
I'm planning to use Stability.ai's image generation models, but I found no plugin for Genkit on the internet. I would be happy to contribute to this matter, but before that, I wanted to ensure there's nothing already done by others.

**Comment by xavidop** [comment]
> I'm planning to use Stability.ai's image generation models, but I found no plugin for Genkit on the internet. I would be happy to contribute to this matter, but before that, I wanted to ensure there's nothing already done by others.

Hi @m-sadegh-sh, I could not find any plugin for Genkit and Stability.ai. You are more than welcome to contribute it in this repo!

**Reply by m-sadegh-sh** [reply]
Perfect. I'll send a PR soon.
